www/site/content/docs/chef-workstation/privacy.md

```diff
@@ -1,30 +1,31 @@
 +++
-title = "Privacy"
+title = "Privacy and Telemetry"
```

Comment:
I think I lost my earlier question here - does this change the slug? If so, there are some link updates in docs and in chef-run.  Maybe we can specify the slug to keep it unchanged? Or maybe this is the time to consider shortlinks, which we haven't done anything with yet.

Reply:
No, this just changes how it appears in the nav-bar. The links are all still good.